composer.json

```diff
@@ -58,7 +58,8 @@
         "itk-dev/serviceplatformen": "^1.5",
         "mglaman/composer-drupal-lenient": "^1.0",
         "os2web/os2web_audit": "^1.0",
-        "os2web/os2web_datalookup": "^2.0",
+        "os2web/os2web_datalookup": "dev-feature/os2web_key as 2.1.0",
```

Comment:
shall we maybe merge os2web_datalookup before merging this PR?
then extra repository mention will not be needed

Reply:
Yes, we should do that. This PR depends on many other PRs. I've requested a review from you on https://github.com/OS2web/os2web_datalookup/pull/13.

Reply:
@stankut, have you seen [my comment](https://github.com/OS2Forms/os2forms/pull/101#discussion_r2111884337)?